src/main/java/rx/observers/TestSubscriber.java

```diff
@@ -83,39 +81,85 @@ public TestSubscriber(Observer<T> delegate, long initialRequest) {
         this.initialRequest = initialRequest;
     }
 
+    /**
+     * Constructs a TestSubscriber which requests Long.MAX_VALUE and delegates events to
+     * the given Subscriber.
+     * @param delegate the subscriber to delegate to.
+     * @throws NullPointerException if delegate is null
+     */
     public TestSubscriber(Subscriber<T> delegate) {
         this(delegate, -1);
     }
 
+    /**
+     * Constructs a TestSubscriber which requests Long.MAX_VALUE and delegates events to
+     * the given Observer.
+     * @param delegate the observer to delegate to.
+     * @throws NullPointerException if delegate is null
+     */
     public TestSubscriber(Observer<T> delegate) {
         this(delegate, -1);
     }
 
+    /**
+     * Constructs a TestSubscriber with an initial request of Long.MAX_VALUE and no delegation.
+     */
     public TestSubscriber() {
```

Comment:
Or for that matter, should all of the public methods in this class (which are not overriding implementations) have `@since 1.1.0` ?